src/components/canned-responses/styles.ts

```diff
@@ -2,6 +2,15 @@ import { css, keyframes } from '@emotion/css';
 import { DesignToken } from '@livechat/design-system-react-components';
 
 const maxWidth = '100%';
+const toPx = (value: number): string => `${value}px`;
+
+const VirtuosoViewportOffsets = {
+  ActionBarHeight: 64,
+  ActionBarMarginTop: 32,
+  ListMarginTop: 16,
+} as const;
+
+export const VIRTUOSO_LIST_HEIGHT = `calc(100vh - ${toPx(Object.values(VirtuosoViewportOffsets).reduce((a, b) => a + b, 0))})`;
```

Comment:
Consider splitting this into two variables to reduce cognitive load and improve readability.

Suggested solution:
```
const virtuosoCalculatedOffset = Object.values(VirtuosoViewportOffsets).reduce((a, b) => a + b, 0))

export const VIRTUOSO_LIST_HEIGHT = `calc(100vh - ${virtuosoCalculatedOffset}
```